examples/example-gateway/clients/quux/quux.go

```diff
@@ -21,18 +21,44 @@
 package quux
 
 import (
-	module "github.com/uber/zanzibar/examples/example-gateway/build/clients/quux/module"
+	"time"
+
+	"github.com/uber/zanzibar/examples/example-gateway/build/clients/quux/module"
 	zanzibar "github.com/uber/zanzibar/runtime"
+
+	"go.uber.org/zap"
 )
 
 // Client is a custom client that does nothing yet
 type Client interface{}
 
-type quux struct{}
+type quux struct {
+	loggers map[string]*zap.Logger
+	metrics map[string]*zanzibar.OutboundCustomMetrics
+}
 
-// NewClient creates a new Quux client
+// NewClient creates a new Quux client, with client method loggers and metrics.
 func NewClient(g *zanzibar.Gateway, deps *module.Dependencies) Client {
-	return &quux{}
+	logger, scope, loggers, metrics := zanzibar.SetupCustomClientLoggersMetrics(
```

Comment:
We shouldn't do custom metrics.

Custom loggers is fine so `zanzibar.SetupClientLoggers(name, methods, deps.Default)` is fine.